ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/420__model_serving.robot

```diff
@@ -127,6 +127,48 @@ Verify Tensorflow Model Via UI
     Should Be Equal As Strings    ${status_code}    200
 
 
+Verify If Model Serving Platforms Is Showing Correct Values In Data Science Project Page
+    [Documentation]    verifes if Cluster setting page of Model Serving Platforms is showing correct value in DS project
+    [Tags]
+    ...       ODS-2574
+    Set Library Search Order    SeleniumLibrary
+    Launch Dashboard   ocp_user_name=${TEST_USER.USERNAME}  ocp_user_pw=${TEST_USER.PASSWORD}
+    ...    ocp_user_auth_type=${TEST_USER.AUTH_TYPE}    dashboard_url=${ODH_DASHBOARD_URL}
+    ...    browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    ...    expected_page=${NONE}    wait_for_cards=${FALSE}
```

Comment:
This opens a new browser but it's never closed. Can you take care of doing that at the end of the test or in the teardown?

Reply:
the suite setup already opens the browser and set the library search order. 

I'd probably move these new tests in a separate suite with its own setup and teardown. Wdyt?